src/shared/forms/models.ts

```diff
@@ -32,9 +32,9 @@ export type ChangeHandler = (e: FormChangeEvent) => void;
 
 export type DirectChangeHandler = (positions: number[], values: any[]) => void;
 
-export type SubmitHandler = (e: FormSubmitEvent) => boolean;
+export type SubmitHandler = (e?: FormSubmitEvent) => boolean;
 
```

Comment:
Just a question - Why is the param nullable? When you're submiting any form, isnt the event always passed?

Reply:
Yes but this type is connected not with form directly. For example there are situations when you would like to use this submit method from useForm without event - you checks validation. Without this nullable parameter i need to pass not needed mock object which simulates event object.